sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -202,17 +206,20 @@ internal open class KlaviyoApiRequest(
      *
      * @returns The string value of the response body, if one was returned
      */
-    fun send(): Status {
+    fun send(beforeSend: (() -> Unit)? = null): Status {
```

Comment:
instead of doing nullable can just default to empty lambda: `beforeSend: (() -> Unit) = {}`